engine/src/main/java/org/terasology/rendering/dag/nodes/InitialPostProcessingNode.java

```diff
@@ -74,23 +71,13 @@
     @Range(min = 0.0f, max = 1.0f)
     private float bloomFactor = 0.5f;
 
-    private SubmersibleCamera activeCamera;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f sunDirection;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f cameraDir;
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector3f cameraPosition;
     @SuppressWarnings("FieldCanBeLocal")
     private Vector3f tint;
 
     public InitialPostProcessingNode(Context context) {
-        backdropProvider = context.get(BackdropProvider.class);
         renderingConfig = context.get(Config.class).getRendering();
         worldProvider = context.get(WorldProvider.class);
-        worldRenderer = context.get(WorldRenderer.class);
-
-        activeCamera = worldRenderer.getActiveCamera();
+        activeCamera = context.get(WorldRenderer.class).getActiveCamera();
 
         DisplayResolutionDependentFBOs displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
         // TODO: see if we could write this straight into a GBUFFER - notice this FBO is used in ShaderParametersHdr
```

Comment:
For the time being just remove the part after the dash. Let's keep this PR focused on the logistic of eliminating shader parameters and let's add the least other changes we can to that.